### PR TITLE
Replace lists of lists with list of Points

### PR DESCRIPTION
**Pull request by BonfaceKilz**

### CODE REVIEW

**Comment by BonfaceKilz** [comment]
@jnduli the idea is to make it to make the coordinates easier to read and interpret ^

File: games/snake.py [hunk 1]
```diff
@@ -1,22 +1,26 @@
 import random
 import curses
 
+from collections import namedtuple
+
+Point = namedtuple('Point', ['x_pos', 'y_pos'])
```
Comment (jnduli) [inline games/snake.py]: Let's use this alternative for this point: https://github.com/jnduli/python_curses_games/blob/3f04025ae26c8d6801b1c4e67f50c0d1467aa150/games/tetris/shapes.py#L3

I think we can place it on a parent class, perhaps Shape.py and import the same into the games

File: games/snake.py [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
+    ]
 
 
 def initial_food(width, height):
-    return [height//2, width//2]
+    return [height // 2, width // 2]
 
 
-class Snake:
+class Snake(object):
```
Comment (jnduli) [inline games/snake.py]: I think the class Snake(object) is a python2 thing. In python 3, it kinda assumes the same. I intend for the project to only support python3

File: games/snake.py [hunk 3]
```diff
@@ -62,13 +66,13 @@ def move_snake(self, key):
             new_head[1] -= 1
         if key in [curses.KEY_RIGHT, ord('l')]:
             new_head[1] += 1
-        self.snake.insert(0, new_head)
+            self.snake.insert(0, new_head)
```
Comment (jnduli) [inline games/snake.py]: This should apply globally to the move_snake method. The extra indent make this only apparent to the last if statement

**Review by jnduli — CHANGES_REQUESTED** [review]

File: games/snake.py (hunk 1 above)
Comment (BonfaceKilz) [reply games/snake.py]: For now, I've moved this to a shapes.py. My intention is that this file will evolve over time as we refactor other things.

File: games/snake.py [hunk 4]
```diff
@@ -104,13 +105,12 @@ def loop(self):
...
-                self.game_window.addch(self.food[0],
-                                       self.food[1], curses.ACS_PI)
+                self.game_window.addch(self.food.y, self.food.x, curses.ACS_PI)
             else:
                 tail = self.snake.pop()
-                self.game_window.addch(tail[0], tail[1], ' ')
-            self.game_window.addch(self.snake[0][0],
-                                   self.snake[0][1], curses.ACS_CKBOARD)
+                self.game_window.addch(tail.y, tail.x, ' ')
+                self.game_window.addch(self.snake[0].y, self.snake[0].x,
```
Comment (jnduli) [inline games/snake.py]: Check indentation

**Review by jnduli — APPROVED** [review]

**Pull request merged**